examples/hybrid/driver.jl

```diff
@@ -154,8 +135,15 @@ if config.parsed_args["check_conservation"]
         energy_surface_change = -p.net_energy_flux_sfc[][]
     end
     energy_radiation_input = -p.net_energy_flux_toa[][]
+
+    energy_net =
+        abs(
+            energy_atmos_change + energy_surface_change -
+            energy_radiation_input,
+        ) / energy_total
+    @info "    Net energy change: $energy_net"
     @test (energy_atmos_change + energy_surface_change) / energy_total ≈
-          energy_radiation_input / energy_total atol = 5 * sqrt(eps(FT))
+          energy_radiation_input / energy_total atol = sqrt(eps(FT))
```

Comment:
Is this equal to `@test (energy_net / energy_total) ≈ 0 atol = sqrt(eps(FT))`? This might be more clear especially as you already have energy_net.